appOPHD/ophd.rc

```diff
@@ -81,7 +81,7 @@ BEGIN
             VALUE "FileDescription", "OutpostHD"
             VALUE "FileVersion", "0.8.7.0"
             VALUE "InternalName", "OutpostHD"
-            VALUE "LegalCopyright", "Copyright � 2015 - 2023, Leeor Dicker"
+            VALUE "LegalCopyright", "Copyright � 2015 - 2023, Leeor Dicker"

```

Comment:
Hmm, curious. There may have been an encoding change here for the copyright character.